manageiq-operator/api/v1alpha1/manageiq_types.go

```diff
@@ -64,8 +64,7 @@ type ManageIQSpec struct {
 	// +optional
 	DatabaseVolumeCapacity string `json:"databaseVolumeCapacity,omitempty"`
 
-	// Flag to indicate if Kafka and Zookeeper should be deployed (default: false)
-	// +optional
```

Comment:
I think we need to keep this comment

Reply:
I've removed the deprecated bit, do you want to keep the `optional` comment too? Technically it will be used by default due to the `true` value but don't think that makes it optional?